lesson_3_6.py

```diff
@@ -0,0 +1,17 @@
+# Пользователь вводит строку текста. Вывести каждое слово с новой строки.
+# ✔ Строки нумеруются начиная с единицы.
+# ✔ Слова выводятся отсортированными согласно кодировки Unicode.
+# ✔ Текст выравнивается по правому краю так, чтобы у самого длинного
+# слова был один пробел между ним и номером строки.
+text = (input("Наберите любой текст: "))
+# строка для теста
+# text = "мой дядя самых честных правил когда не в шутку занемог"
+arr = text.split(" ")
```

Comment:
Если хотите разделить по пробелам, то лучше ничего не передавать методу `split`.
```suggestion
arr = text.split()
```